build/webvr-polyfill.js

```diff
@@ -2367,9 +2422,10 @@ function CardboardVRDisplay() {
   // Set the correct initial viewer.
   this.deviceInfo_.setViewer(this.viewerSelector_.getCurrentViewer());
 
-  this.rotateInstructions_ = new RotateInstructions();
-
-  this.injectPresentModeCssClass_();
+  if (!WebVRConfig.ROTATE_INSTRUCTIONS_DISABLED) {
+    this.rotateInstructions_ = new RotateInstructions();
+    this.injectPresentModeCssClass_();
```

Comment:
This should happen regardless of whether or not instructions are disabled. It's a fix for fullscreen mode.